posthog/queries/paths/paths_actors.py

```diff
@@ -25,6 +27,11 @@ class PathsActors(Paths, ActorBaseQuery):  # type: ignore
         other path item between start and end key.
     """
 
+    def __init__(self, filter: PathFilter, team: Team, funnel_filter: Optional[Filter] = None,) -> None:
+        super().__init__(filter, team, funnel_filter)
```

Comment:
I see the issue now. The same must apply in funnels as well, since `ActorBaseQuery.__init__` isn't called there either?

The problem isn't that it's overridden, but never called, based on MRO (method-resolution-order).

Hmm, just calling `__init__` for ActorBaseQuery in all these cases wouldn't work, since then the extended filter would be in `ActorBaseQuery`, and not accessed via `self._filter` (which will go first to `Paths._filter`).

Hmm, I see. This is annoying. Several places where this is borked:

1. Correlations ( `FunnelCorrelationActors` never calls `super()`)
2. All funnels: FunnelTrends, FunnelUnordered, FunnelStrict, etc. etc.

For all of these,

Reply:
Ah yeah thats frustrating. 
I'll take another look and see if there is some other cleverness that could be done

Reply:
Yeah, not too happy with this either, since it's going to be very easy to forget to do this in any new class we write

Reply:
The only "tidy" option I can think of is to do this in the API controller instead. Will still have to be remembered each time but at least it is done in one place...

Reply:
Makes sense to me, it's in place with other filter updates